app/controllers/SignInController.scala

```diff
@@ -42,6 +43,17 @@ abstract class SignInController(val applicationClient: ApplicationClient, cacheC
       }
   }
 
+  def sdipPresent = CSRUserAwareAction { implicit request =>
```

Comment:
there is already a method present - we display index for sign in,
this sdipPresent is continuation of sign in but for the sdip path. We may refactor the naming later